first.c

```diff
@@ -0,0 +1,112 @@
+
+#include <stdio.h>
+#include <ctype.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+
+void removeWhitespaces(char *exp)
+{
+    char *p = exp;
+    char *q = exp;
+    while (*q != '\0')
+    {
+        if (!isspace(*q))
+        {
+            *p++ = *q;
+        }
+        q++;
+    }
+    *p = '\0';
+};
+
+bool checkValidity(char *exp)
+{
+    char oprs[4] = {'+', '-', '*', '/'};

```

Comment:
give meaningful names to variable as well as to methods. Take care at other places

Reply:
I will update the variable and method names to make them more meaningful and take care of other areas as well.